### ISSUE TITLE
[BUG] Links are hard to read in dark mode. 

### ISSUE THREAD

**Issue by tinashekupfuwa** [michaeldera/edliz#71]
**Describe the bug**
Blue links are hard to read when you are in dark mode. The color contrast is really bad. 

**To Reproduce**
1. Go to 'Reading View'
2. Navigate to Chapter 36
3. Change to Night Mode
4. See error (Hard-to-read text)

**Expected behavior**
Color contrast should be better.

**Screenshots**
If possible include screenshots to help explain your problem.

**Platform: Where did you see this bug (please remove those what does not apply):**
 - Google Chrome on Mac 
 - Google Chrome on iOS 

**Additional context**
![edliz azurewebsites net_](https://user-images.githubusercontent.com/26183787/60510916-f3dd2d80-9cd0-11e9-8d8a-6744bc67456a.png)

**Comment by michaeldera** [comment]
Would you like to work on it?

**Comment by tinashekupfuwa** [comment]
> Would you like to work on it?

Sure!
